formstack-consents/README.md

```diff
@@ -1,20 +1,39 @@
 # Formstack Consents Lambda
 
-Upon a submission to one of the Formstack forms specified in com.gu.identity.formstackconsents.Newsletter, webhooks set up via Formstack's UI send a POST request to /consent, triggering this lambda via an API Gateway. Once the email address and form ID have been retrieved from the Formstack submission, a POST request is then made to Identity, triggering a confirmation email to be sent to that user.
+Used to trigger a consent email from IDAPI upon submssion of a [Formstack](https://guardiannewsandmedia.formstack.com) newsletter signup form or marketing consent form via a webhook.
 
-When the user clicks the link in the email, an identity account will be created for them and they will be signed up to the newsletter. 
+Example Formstack newsletter signup forms used for [Professional Networks]:(https://www.theguardian.com/guardian-professional)
+- [Teacher Newsletter](https://www.theguardian.com/teacher-network/2018/mar/21/guardian-teacher-network-newsletter-sign-up)
+- [Society Newsletter](https://www.theguardian.com/society/2018/jun/20/society-weekly-email-newsletter-sign-up)
 
-# Running Serverless Application Model locally
+
+## How the lambda works
```

Comment:
For describing implementation detail at this (granular) level, my preference would be to use in-line comments, since it reduces the chance of the documentation coming out of sync with the code. That said, it's still really useful and should help other developers working on this lambda in the future.

Reply:
I've taken this out as it was an attempt to expound on the intro that has gone back in.